src/components/BannerHighlight/BannerHighlight.vue

```diff
@@ -12,8 +12,8 @@
             <!-- prompt -->
 
             <div class="prompt-container">
-                <p class="prompt" v-html="prompt" />
-                <span class="arrow">Arrow</span>
+                <span class="prompt" v-html="prompt" />
+                <arrow class="arrow" />
```

Comment:
Can we call this <svg-arrow> and SvgArrow, thanks!

FYI This is an example of something I'd love to just be able to change myself easily.

Reply:
updated